modules/protect/math-fallback.php

```diff
@@ -124,8 +124,15 @@ static function math_form() {
 			?>
 			<div style="margin: 5px 0 20px;">
 				<strong><?php esc_html_e( 'Prove your humanity:', 'jetpack' ); ?> </strong>
-				<?php echo $num1 ?> &nbsp; + &nbsp; <?php echo $num2 ?> &nbsp; = &nbsp;
-				<input type="input" name="jetpack_protect_num" value="" size="2" />
+				<label for="sum">
+					<?php echo $num1; ?>
+					<span class="accessibility-visually-hidden">__(plus)</span>
```

Comment:
This will render `__(plus)` as text. Not sure how a screen reader would read this but it's confusing. The right syntax for the text is
```php
<?php esc_html_e( 'plus' ) ?>
```